packages/grid/_modules_/grid/components/GridRow.tsx

```diff
@@ -33,7 +33,12 @@ export const GridRow = (props: GridRowProps) => {
   const publish = React.useCallback(
     (eventName: string) => (event: React.MouseEvent) => {
       // Ignore portal
-      if (!event.currentTarget.contains(event.target as HTMLElement)) {
+      // The target is not an element when triggered by a Select inside the cell
+      // See https://github.com/mui-org/material-ui/issues/10534
+      if (
+        (event.target as any).nodeType === 1 &&
+        !event.currentTarget.contains(event.target as Element)
```

Comment:
BTW I thought I put a comment about this one, or is it on another PR and we are repeating it again?

Reply:
We're no repeating again. This PR is stacked on top of #1772, which is where you added your comment. #1772 has to be reviewed first.